recipes/dask_groupby/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "dask_groupby" %}
+{% set version = "0.1.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dask_groupby-{{ version }}.tar.gz
+  sha256: a3766df3775caf44be2561b50736e9634f6326b2e4c9c615f5a0728de3a554e1
+
+build:
+  number: 0
+  skip: true   # [py<37]
```

Would be better to make this noarch python:
```suggestion
  noarch: python
```

You can just unconditionally depend on `importlib-metadat` and `typing-extensions`.